src/components/BatchSearchTable.vue

```diff
@@ -445,7 +451,9 @@ export default {
     getProjectsNames(item) {
       return item.projects?.map((project) => project.name).join(', ') ?? ''
     },
-
+    isSelected(item) {
```

Comment:
If you switch to a boolean prop for `eq`, the method `isSelected` should be a computed property.